pkg/operation/common/utils.go

```diff
@@ -471,6 +473,73 @@ func DeleteReserveExcessCapacity(ctx context.Context, k8sClient client.Client) e
+		&extensionsv1beta1.Ingress{ObjectMeta: metav1.ObjectMeta{Name: "kibana", Namespace: namespace}},
+		&corev1.Secret{ObjectMeta: metav1.ObjectMeta{Name: "kibana-basic-auth", Namespace: namespace}},
+		&corev1.ServiceAccount{ObjectMeta: metav1.ObjectMeta{Name: "elasticsearch-logging", Namespace: namespace}},
+		&corev1.Service{ObjectMeta: metav1.ObjectMeta{Name: "elasticsearch-logging", Namespace: namespace}},
+		&corev1.Service{ObjectMeta: metav1.ObjectMeta{Name: "fluentd-es", Namespace: namespace}},
+		&corev1.Service{ObjectMeta: metav1.ObjectMeta{Name: "kibana-logging", Namespace: namespace}},
+		&appsv1.StatefulSet{ObjectMeta: metav1.ObjectMeta{Name: "fluentd-es", Namespace: namespace}},
+		&appsv1.StatefulSet{ObjectMeta: metav1.ObjectMeta{Name: "elasticsearch-logging", Namespace: namespace}},
+		&corev1.PersistentVolumeClaim{ObjectMeta: metav1.ObjectMeta{Name: "elasticsearch-logging-elasticsearch-logging-0", Namespace: namespace}},
+		&corev1.PersistentVolumeClaim{ObjectMeta: metav1.ObjectMeta{Name: "fluentd-fluentd-es-0", Namespace: namespace}},
+		&corev1.PersistentVolumeClaim{ObjectMeta: metav1.ObjectMeta{Name: "fluentd-fluentd-es-1", Namespace: namespace}},
+		&corev1.PersistentVolumeClaim{ObjectMeta: metav1.ObjectMeta{Name: "fluentd-fluentd-es-2", Namespace: namespace}},
+		&corev1.PersistentVolumeClaim{ObjectMeta: metav1.ObjectMeta{Name: "fluentd-fluentd-es-3", Namespace: namespace}},
+		&corev1.PersistentVolumeClaim{ObjectMeta: metav1.ObjectMeta{Name: "fluentd-fluentd-es-4", Namespace: namespace}},
+		&corev1.PersistentVolumeClaim{ObjectMeta: metav1.ObjectMeta{Name: "fluentd-fluentd-es-5", Namespace: namespace}},
+		&corev1.PersistentVolumeClaim{ObjectMeta: metav1.ObjectMeta{Name: "fluentd-fluentd-es-6", Namespace: namespace}},
+		&corev1.PersistentVolumeClaim{ObjectMeta: metav1.ObjectMeta{Name: "fluentd-fluentd-es-7", Namespace: namespace}},
+		&corev1.PersistentVolumeClaim{ObjectMeta: metav1.ObjectMeta{Name: "fluentd-fluentd-es-8", Namespace: namespace}},
+		&corev1.PersistentVolumeClaim{ObjectMeta: metav1.ObjectMeta{Name: "fluentd-fluentd-es-9", Namespace: namespace}},
+		&networkingv1.NetworkPolicy{ObjectMeta: metav1.ObjectMeta{Name: "allow-elasticsearch", Namespace: namespace}},
+		&networkingv1.NetworkPolicy{ObjectMeta: metav1.ObjectMeta{Name: "allow-to-elasticsearch", Namespace: namespace}},
+		&networkingv1.NetworkPolicy{ObjectMeta: metav1.ObjectMeta{Name: "allow-fluentd", Namespace: namespace}},
+		&networkingv1.NetworkPolicy{ObjectMeta: metav1.ObjectMeta{Name: "allow-kibana", Namespace: namespace}},
+		&vpa_api.VerticalPodAutoscaler{ObjectMeta: metav1.ObjectMeta{Name: "elasticsearch-logging-vpa", Namespace: namespace}},
+	}
+
+	for _, resource := range resources {
+		if err := client.IgnoreNotFound(k8sClient.Delete(ctx, resource)); err != nil {
+			return err
+		}
+	}
+
+	ds := &appsv1.DaemonSet{ObjectMeta: metav1.ObjectMeta{Name: "fluent-bit", Namespace: namespace}}
```

Comment:
No need to specify anything here as you already pass key to `Get`.

```suggestion
	ds := &appsv1.DaemonSet{}
```